docs/CODE_GUIDELINES.md

```diff
@@ -598,17 +663,22 @@ The `Log` function uses the [fmt library](http://fmtlib.net/) for formatting log
 
 ✅ Good:
 ```cpp
-CLog::Log(LOGDEBUG, "Window size: {}x{}", width, height);
+uint32_t width{1920};
+uint32_t height{1080};
+std::string_view name{"List of foobar"};
```

Comment:
might want to make this constexpr in line with the suggestions for string views?

Reply:
what do you mean?

Reply:
the string_view example that you added uses `constexpr std::string_view` for a string constant

Reply:
not same thing that example was used on static const vars on a unnamed namespace
instead this code is intended within a method
CLog::Log cannot be used outside of a method

Reply:
was just an idea though, if you don't want to add it here that's also fine for me.

Reply:
side-note: it's good practice nevertheless to make  variables `const` or `constexpr` if they should not change, even for local variables within methods.

Reply:
agreed but since variables here are not really needed for the example i removed them all